src/dos/meson.build

```diff
@@ -39,10 +39,12 @@ libdos_sources = files(
     'program_mount.cpp',
     'program_mount_common.cpp',
     'program_mousectl.cpp',
+    'program_move.cpp',
     'program_placeholder.cpp',
     'program_rescan.cpp',
     'program_serial.cpp',
     'program_setver.cpp',
+    'program_subst.cpp',
```

Comment:
You need to update the Visual Studio project files too:

https://github.com/dosbox-staging/dosbox-staging/blob/main/vs/dosbox.vcxproj
https://github.com/dosbox-staging/dosbox-staging/blob/main/vs/dosbox.vcxproj.filters

Once you've done that, the VS builds should start passing. The MSYS2 failure is a different issue, unrelated to your change, so please ignore that.

Reply:
Those files are a mess.  I think I added it to the right place but lets see if the builds pass.  Is there a reason we can't just use Meson to generate Visual Studio files so we don't have to maintain these?

Reply:
Dunno. It's not too hard to update them, and you don't need to do that very often.

If you can come up with a better solution I'm not against it, but you'll need to fully test it, make the CI loops pass, etc.

Reply:
Fair enough.  I was just curious.  I'm probably not the right person to come up with a better solution.  I hate fussing with build systems... especially Windows build systems.

Reply:
I'm the same, man 😄